examples/neuralop/checkpoint_FNO_darcy.py

```diff
@@ -0,0 +1,145 @@
+"""
+Training a TFNO on Darcy-Flow
+=============================
+
+In this example, we demonstrate how to use the small Darcy-Flow example we ship with the package
+to train a Tensorized Fourier-Neural Operator
+"""
+
+import sys
+
+# %%
+#
+import paddle
+
+from ppsci.contrib.neuralop import H1Loss
+from ppsci.contrib.neuralop import LpLoss
+from ppsci.contrib.neuralop import Trainer
+from ppsci.contrib.neuralop.datasets import load_darcy_flow_small
+from ppsci.contrib.neuralop.models import TFNO
+from ppsci.contrib.neuralop.training import CheckpointCallback
+from ppsci.contrib.neuralop.utils import count_model_params
+
+device = "cpu"
```

Comment:
paddle与torch不同，torch的Tensor、Module默认创建在CPU上，而paddle默认创建在当前设备（一般是GPU）上，所以比较好的方式是在开头直接把paddle的当前设备设置为cpu，这样后续所有的操作都不需要再通过`device`去设置了。
```suggestion
device = "cpu"
paddle.device.set_device(device)
```